lib/s3.py

```diff
@@ -1,12 +1,37 @@
+import os
 import boto3
+from botocore.client import Config
+from botocore.exceptions import ClientError
 from lib.logger import logger
 def upload_file_to_s3(bucket: str, filename: str):
     """
     Generic upload helper for s3. Could be moved over to helpers folder...
     """
+    s3_url = os.getenv('S3_ENDPOINT')
+    access_key = os.getenv('AWS_ACCESS_KEY_ID')
```

Comment:
In timpani, we did access using the roles, vs access key, so maybe this is not needed?

Reply:
I'll let @sonoransun weigh in on this - this is how I've _usually_ done this sort of thing, but maybe its not needed?

Reply:
FWIW I would _prefer_ leaving it this way as currently written as it interfaces nicely with minio locally - no biggie if that's a problem though!